vo_new/app/login/page.tsx

```
@@ -0,0 +1,189 @@
+          {isSubmitting ? "Signing in..." : "Sign In"}
+        </Button>
+
+        <div className="text-center text-sm text-gray-600 dark:text-gray-400">
+          <a href="/forgot-password" className="hover:underline">
+            Forgot password?
+          </a>
+        </div>
+
+        <div className="text-center text-sm">
+          Not a member?{" "}
+          <a href="/signup" className="text-primary hover:underline font-medium">
```

ESLint

[error] Do not use an `<a>` element to navigate to `/signup/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)